arm-template-retrieve-azure-storage-access-keys/azuredeploy.json

```diff
@@ -0,0 +1,186 @@
+            "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_RAGRS",
+                "Standard_ZRS",
+                "Premium_LRS",
+                "Premium_ZRS",
+                "Standard_GZRS",
+                "Standard_RAGZRS"
+            ],
+            "metadata": {
+                "description": "Sku on which to run the Azure Storage account."
+            }
+        },
+        "storageAccountKind": {
+            "type": "string",
+            "defaultValue": "StorageV2",
+            "allowedValues": [
+                "Storage",
+                "StorageV2",
+                "BlobStorage",
+                "FileStorage",
+                "BlockBlobStorage"
+            ],
+            "metadata": {
+                "description": "Indicates the type of storage account."
+            }
+        },
+        "storageAccountContainerName": {
+            "type": "string",
+            "defaultValue": "my-container",
+            "metadata": {
+                "description": "Set the name of the container to create in the Storage account."
+            }
+        },
+        "storageConnectionName": {
+            "type": "string",
+            "defaultValue": "[concat('storageconnection', uniqueString(resourceGroup().id))]",
+            "metadata": {
+                "description": "Name of the Logic Apps API connection used to connect to the Azure Storage account."
+            }
+        },
+        "logicAppName": {
+            "type": "string",
+            "defaultValue": "[concat('logicapp', uniqueString(resourceGroup().id))]",
+            "metadata": {
+                "description": "Name of the Logic App."
+            }
+        },
+        "logicAppPollingIntervalInMinutes": {
+            "type": "int",
+            "defaultValue": 30,
+            "metadata": {
+                "description": "The polling interval used to check for items on the Storage account."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location where resources reside."
+            }
+        }
+    },
+    "variables": {
+        "storageAccountId": "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName'))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Web/connections",
+            "apiVersion": "2018-07-01-preview",
+            "name": "[parameters('storageConnectionName')]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName'))]"
+            ],
+            "properties": {
+                "api": {
+                    "id": "[concat(subscription().id,'/providers/Microsoft.Web/locations/', parameters('location'), '/managedApis/azureblob')]"
```

Comment:
remove concat and use resourceId

resourceId('Microsoft.Web/locations/managedApis', parameters('location'), 'azureblob')

Reply:
This doesn't work for the managed APIs.
The concat is also how this is described in the official documentation, f.e. https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-azure-resource-manager-templates-overview#connection-resource-definitions.
Also this is the same in other Logic App templates in the quickstart samples.